packages/esm-patient-chart-app/src/visit/visits-widget/visit-context/visit-context-header.component.tsx

```diff
@@ -34,6 +39,9 @@ const VisitContextHeader: React.FC<VisitContextHeaderProps> = ({ patientUuid })
     });
   };
 
+  if (!showVisitContextHeader) {
+    return <></>;
```

Comment:
nit: `null` is also a valid React render result.

Reply:
With the added benefit that `return null`:
- Is more performant as it doesn't create any objects or elements as opposed to `return <></>` which creates a React fragment even though it doesn't get render anything in the DOM.
- Uses less memory because `null` is a primitive value whereas `<></>` creates a React fragment.
- Communicates more intent as it's the idiomatic way of saying "don't render anything" in React.